politeiawww/api/v1/v1.go

```diff
@@ -1094,3 +1095,8 @@ type ProposalsStatsReply struct {
 	NumOfPublic          int `json:"numofpublic"`          // Counting number of public proposals
 	NumOfAbandoned       int `json:"numofabandoned"`       // Counting number of abandoned proposals
 }
+
+// ProposalAccessTimeReply returns an array of all proposal access replies
```

Comment:
I don't think the way you're handling the `PUT` request is RESTful. You're telling the server to update a record, but you're not telling the server what to update the record to.  i.e. you're letting the server create the timestamp instead of creating the timestamp on the client and sending it to the server.  Doing it this way also means that the `PUT` request isn't idempotent.

I think the `PUT` request should be structured like this:

```
// UserProposalAccessTime is used to update the user's access time of the
// specified proposal.
type UserProposalAccessTime struct {
	Token      string `json:token`      // Proposal censorship token
	AccessTime int64  `json:accesstime` // UNIX timestamp of last time user accessed proposal
}

// UserProposalAccessTimeReply is used to reply to the UserProposalAccessTime
// command.
type UserProposalAccessTimeReply struct{}
```

Where `Token` is taken from the path and `AccessTime` is included in the request body.